apps/api/src/domain/domainModelValidators/__tests__/contextValidators/buildEdgeConnectionContextValidatorTestCases/freeMultiline.contextValidator.TestCase.ts

```diff
@@ -0,0 +1,92 @@
+import { InternalError } from '../../../../../lib/errors/InternalError';
+import { PartialDTO } from '../../../../../types/partial-dto';
+import InvalidLineInFreeMultilineError from '../../../../domainModelValidators/errors/context/InvalidLineInFreeMultilineError';
+import MissingLineContextError from '../../../../domainModelValidators/errors/context/MissingLineContextError';
+import { FreeMultilineContext } from '../../../../models/context/free-multiline-context/free-multiline-context.entity';
+import { EdgeConnectionContextType } from '../../../../models/context/types/EdgeConnectionContextType';
+import { freeMultilineContextValidator } from '../../../contextValidators/freeMultilineContext.validator';
+import NullOrUndefinedEdgeConnectionContextDTOError from '../../../errors/context/NullOrUndefinedEdgeConnectionContextDTOError';
+import { ContextModelValidatorTestCase } from '../types/ContextModelValidatorTestCase';
+import createInvalidContextErrorFactory from './utilities/createInvalidContextErrorFactory';
+
+const validDTO: PartialDTO<FreeMultilineContext> = {
+    type: EdgeConnectionContextType.freeMultiline,
+
+    lines: [
```

Comment:
A test helper to generate valid lines is starting to look useful. Do this next time you seed some linear structure test data.